conversions/celsius_to_fahrenheit.py

```diff
@@ -1,34 +1,28 @@
 """ Convert temperature from Celsius to Fahrenheit """
 
 
-def celsius_to_fahrenheit(celsius):
+def celsius_to_fahrenheit(celsius: float) -> float:
     """
-    Convert a given value from Celsius to Fahrenheit
+    Convert a given value from Celsius to Fahrenheit and round it to 2 decimal places.
 
-    >>> celsius_to_fahrenheit(-40)
+    >>> print(celsius_to_fahrenheit(-40))
```

Comment:
Please remove these print statements and allow doctest to directly evaluate the return values of the function call.